bigtop-deploy/vm/vagrant-puppet-vm/Vagrantfile

```diff
@@ -112,7 +113,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
       bigtop.vm.provider :virtualbox do |vb|
         vb.customize ["modifyvm", :id, "--memory", CONF['memory_size']]
-	vb.customize ['modifyvm', :id, '--cpus', CONF['number_cpus']]
+        vb.customize ['modifyvm', :id, '--cpus', CONF['number_cpus']]
```

Comment:
This appears to be just a whitespace change - if you'll be fixing the comment above, can you please remove this spurious change as well?